dashboard/app/controllers/plc/enrollment_task_assignments_controller.rb

```diff
@@ -0,0 +1,35 @@
+class Plc::EnrollmentTaskAssignmentsController < ApplicationController
+  before_action :set_plc_enrollment_task_assignment, only: [:show, :destroy]
+
+  # GET /plc/enrollment_task_assignments
+  # GET /plc/enrollment_task_assignments.json
+  def index
+    @plc_enrollment_task_assignments = Plc::EnrollmentTaskAssignment.all
```

Comment:
Should this be filtered by the current user?  Otherwise this will return EnrollmentTaskAssignments for all users.